### DISCUSSION TITLE
Dockerfile Not Working...  Fixed

### DISCUSSION THREAD

**Discussion by dwilliamhouston** [tobychui/arozos#81] [General]
There was a change between version 1.112 and 1.113 in ArozOS where System directory moved which caused the docker file to break.  Since I didnt have permissions to change the file in Saren-Arterius github share I have forked it and made changes at https://github.com/dwilliamhouston/aroz-dockerize.  This now works with latest version of Arozos (1.116) as well as version before 1.116.

**Comment by tobychui** [comment]
Hi @dwilliamhouston, thanks so much for your contribution :DDD

I am sorry as I don't really know much about Docker and docker-files. But I guess you can try creating a PR to Saren-Arterius's repo. If you are willing to be the maintainer for the aroz-docker build, feel free to let me know and I will update the README to include your repo as well.

**Reply by dwilliamhouston** [reply]
I have sent the PR over to Saren and will see if he responds but his repo hasnt been updated in 8 months so not sure if he is doing anything anymore.  If not then I will let you know and I can maintain it.

**Reply by tobychui** [reply]
Ok sure, looking forward to your updates :)

**Reply by Saren-Arterius** [reply]
Hello there, I see that your PR has closed without reason. Please notify if the PR is ready.

**Reply by dwilliamhouston** [reply]
So I have added support to do build of containers now and they are stored here:

https://github.com/dwilliamhouston/aroz-dockerize/pkgs/container/aroz-dockerize

There are currently builds for linux/amd64 and linux/arm64.  Let me know if you want any others.
